content/docs/releases/release-notes/release-notes-1.16.md

```diff
@@ -7,12 +7,31 @@ cert-manager 1.16 includes various improvements to the metrics in the cert-manag
 
 ## Breaking changes
 
-1. Venafi Issuer may fail to renew Certificates if the duration conflicts with the CA minimum / maximum duration policy in Venafi.
-2. Venafi Issuer may fail to renew Certificates if the issuer has been configured for TPP with username-password authentication.
-3. Helm schema validation may reject your existing Helm values files if they contain typos or unrecognized fields.
+1. Helm schema validation may reject your existing Helm values files if they contain typos or unrecognized fields.
+   For more details, refer to the [Helm](#helm) section below.
+1. Venafi Issuer may fail to renew certificates if the requested duration conflicts with the CA’s minimum or maximum policy settings in Venafi.
+   For more details, refer to the [Venafi Issuer](#venafi-issuer) section below.
+1. Venafi Issuer may fail to renew Certificates if the issuer has been configured for TPP with username-password authentication.
+   For more details, refer to the [Venafi Issuer](#venafi-issuer) section below.
 
 ## Themes
 
+### Helm
+
+The Helm chart now includes a JSON schema which will validate the values that you supply when installing the chart.
+This will help you to get your Helm values right first time.
+It will alert you to typos and unrecognized fields in your existing Helm values files.
+And it will make it easier for the cert-manager maintainers to maintain the Helm chart,
+avoiding typos and mistakes in the default values file.
+
+> ⚠️ Helm schema validation may reject your existing Helm values files if they contain typos or unrecognized fields.
+> You can use `helm template cert-manager --repo https://charts.jetstack.io --version 1.16.x --values values.cert-manager.yaml`
```

Comment:
suggestion: Having the command in just backticks like this means it gets split across lines:

<img width="751" alt="Screenshot 2024-09-27 at 18 31 35" src="https://github.com/user-attachments/assets/8a4f1627-c046-4101-bb2e-308f3484810d">

That makes it harder to copy and I think a lot of people upgrading will want to copy it!

I'd suggest having it in triple backticks instead so it's rendered as a code block:

```suggestion
>   ```bash
>   helm template cert-manager --repo https://charts.jetstack.io --version 1.16.x --values values.cert-manager.yaml
>   ```
```

(the suggestion might not work as-is, but you get the idea)

Reply:
@SgtCoDFish Done. And added an example of an error.

@jsoref You're suggested command would only work for your specific sub-chart use-case, right? For general users it is better to state the repo URL (otherwise they'd need to run `helm repo update` first) and it's better to state the version otherwise they may not be testing against the cert-manager 1.16 release. 

I wrote `--version 1.16.x` deliberately, because I thought it would continue to work for future 1.16 patch releases.
It does not work with 1.16 pre-releases without an additional `--devel` flag.
So I've added an explicit version variable instead which should be updated with future patch releases.